solr/core/src/java/org/apache/solr/core/SolrResourceLoader.java

```diff
@@ -814,6 +815,7 @@ public Path getInstancePath() {
         new Class<?>[] {
           // DO NOT ADD THINGS TO THIS LIST -- ESPECIALLY THINGS THAT CAN BE CREATED DYNAMICALLY
           // VIA RUNTIME APIS -- UNTIL CAREFULLY CONSIDERING THE ISSUES MENTIONED IN SOLR-8311
+          CircuitBreaker.class,
```

Comment:
Any objections to add CircuitBreaker to the list of SolrCoreAware classes? Solr's home grown IOC sucks but that's what we have...